src/components/MapView/MapView.website.tsx

```diff
@@ -112,11 +112,15 @@ const MapView = forwardRef<MapViewHandle, ComponentProps>(
                 return;
             }
 
+            // Avoid animating the naviagtion to the same location
+            const shouldAnimate = prevUserPosition.longitude !== currentPosition.longitude || prevUserPosition.latitude !== currentPosition.latitude;
+
             mapRef.flyTo({
                 center: [currentPosition.longitude, currentPosition.latitude],
                 zoom: CONST.MAPBOX.DEFAULT_ZOOM,
+                animate: shouldAnimate,
             });
-        }, [currentPosition, userInteractedWithMap, mapRef, shouldPanMapToCurrentPosition]);
```

Comment:
This is my only question - do we not need `userInteractedWithMap` anymore since we're only using `shouldPanMapToCurrentPosition`?

Reply:
great!